geoips/create_plugin_registry.py

```diff
@@ -0,0 +1,76 @@
+import yaml
+import numpy as np
+from importlib import metadata, resources, util
+import os
+import sys
+
+plugins = {}
+
+def get_entry_point_group(group):
+    """Get entry point group."""
+    if sys.version_info[:3] >= (3, 10, 0):
+        return metadata.entry_points(group=group)
+    else:
+        return metadata.entry_points()[group]
+
+def main():
+    plugin_packages = get_entry_point_group("geoips.plugin_packages")
+    print(plugin_packages)
+    for pkg in plugin_packages:
+        package = pkg.value
+        print("package == " + str(package))
+        pkg_plugin_path = resources.files(package) / "plugins"
+        yaml_files = pkg_plugin_path.rglob("*.yaml")
+        python_files = pkg_plugin_path.rglob("*.py")
+        if package != "geoips":
```

Comment:
A comment describing what this if statement is doing would be helpful. Why do this for plugin packages, but not for geoips?

Reply:
That was actually unnecessary. Just removed that if statement.